packages/webcert/src/feature/certificate/UnifiedEdit/UeInteger/UeInteger.tsx

```diff
@@ -42,12 +42,14 @@ const UeInteger: React.FC<Props> = ({ question, disabled }) => {
   }
 
   const handleNumberOnChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const inputValue = event.target.value
-
+    let inputValue = event.target.value
     if (/^[a-zA-Z]+$/.test(inputValue)) {
-      return null
+      return
+    }
+    if (inputValue.indexOf('-') > 0) {
+      inputValue = inputValue.replace('-', '')
```

Comment:
This will only replace the first occurance of '-', if someone pastes in a number with more than onr '-' you'll get the same error.  you'll probably need something like replace(/-/g,'') instead.

Reply:
The above regex `/^[a-zA-Z]+$/.test(inputValue)` could be changed to `/[^\d]/.test(inputValue)` to only allow numerical.

Reply:
hmm but we also want to handle negative numbers

Reply:
Alright, we can include that too, but it becomes a bit more readable if we flip the expression `!/^-?[\d]+$/.test(inputValue)`